### PR TITLE
French translation for the `descent-override` CSS descriptor reference

### PR DESCRIPTION
**Pull request by audrasjb**
This PR adds a French translation for the `descent-override` CSS descriptor reference.
See also #1752 

Please note that I asked a question about the best practice for `cssxref` references translation here: https://matrix.to/#/!pGdhkeiUTYDIWuskqX:mozilla.org/$bdvXKWkU1RyybuAjp0JmacC9RK8I2r7sF6yuhtIIFZM?via=mozilla.org&via=t2bot.io&via=matrix.org

### CODE REVIEW

File: files/fr/web/css/@font-face/descent-override/index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+<pre class="brush: css">descent-override: normal;
+descent-override: 90%;</pre>
+
+<h3 id="values">Valeurs</h3>
+
+<dl>
+  <dt><code>normal</code></dt>
+  <dd>La valeur par défaut. La mesure du jambage inférieur est obtenue à partir du fichier de police.</dd>
+  <dt><code>&lt;percentage&gt;</code></dt>
+  <dd>Une valeur exprimée en {{cssxref("&lt;percentage&gt;", "&lt;pourcentage&gt;")}}.</dd>
```
Comment (SphinxKnight) [inline files/fr/web/css/@font-face/descent-override/index.html:22]: Same as https://github.com/mdn/translated-content/pull/1752/files#r678016933 :)

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
shamelessly cloning feedback from #1752 :)

File: files/fr/web/css/@font-face/descent-override/index.html (hunk 1 above)
Comment (audrasjb) [reply files/fr/web/css/@font-face/descent-override/index.html]: Fixed in https://github.com/mdn/translated-content/pull/1753/commits/aea8cad8466742d4670b37aa38a80c1ac2149371

**Review by SphinxKnight — APPROVED** [review]

**Comment by SphinxKnight** [comment]
Thanks :)

**Pull request merged**